common/static/common/js/spec/components/.gitignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
@muhammad-ammar I don't like this idea, because it will make it harder for people to notice that they haven't added new JS files to Git. I recommend either converting this file to JavaScript, or creating a top level coffee directory. My preference would be the former if possible.